metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -274,11 +310,6 @@ public void run() throws Exception {
                 new EventQueue.DeadlineFunction(deadline),
                 new PollEvent());
         }
-
```

Comment:
Why are we not handling exceptions in other events?

Reply:
The base MigrationEvent will do the error logging. I was just removing some redundant overrides.